<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="text">
     <string>Welcome to use Hospital System</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>646</width>
      <height>365</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>646</width>
      <height>365</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRandom_Generate"/>
    <addaction name="separator"/>
    <addaction name="actionInput"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOne_Wait_Time"/>
    <addaction name="separator"/>
    <addaction name="actionAverage_Wait_Time"/>
    <addaction name="separator"/>
    <addaction name="actionLongest_Wait_Time"/>
   </widget>
   <widget class="QMenu" name="menuInquire">
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Inquire</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIs_Full"/>
    <addaction name="separator"/>
    <addaction name="actionIs_Empty"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout_Program"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_me"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuInquire"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRandom_Generate">
   <property name="text">
    <string>Random Generate</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionIs_Full">
   <property name="text">
    <string>Is Full</string>
   </property>
  </action>
  <action name="actionIs_Empty">
   <property name="text">
    <string>Is Empty</string>
   </property>
  </action>
  <action name="actionAverage_Wait_Time">
   <property name="text">
    <string>Average Wait Time</string>
   </property>
  </action>
  <action name="actionAbout_Program">
   <property name="text">
    <string>About Program</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="text">
    <string>About Me</string>
   </property>
  </action>
  <action name="actionOne_Wait_Time">
   <property name="text">
    <string>One Wait Time</string>
   </property>
  </action>
  <action name="actionLongest_Wait_Time">
   <property name="text">
    <string>Longest Wait Time</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
